history/taskana-simplehistory-provider/src/test/java/acceptance/events/task/CreateHistoryEventOnTaskTransferAccTest.java

```diff
@@ -35,23 +38,32 @@ class CreateHistoryEventOnTaskTransferAccTest extends AbstractAccTest {
   @WithAccessId(user = "admin")
   @TestFactory
   Stream<DynamicTest> should_CreateTransferredHistoryEvent_When_TaskIsTransferred() {
-    List<Triplet<String, String, Consumer<String>>> testCases =
+    List<Quadruple<String, String, String, Consumer<String>>> testCases =
         List.of(
-            Triplet.of(
+            Quadruple.of(
                 "Using WorkbasketId",
```

Comment:
add description clarifying that TKI:000000000000000000000000000000000003 has no attachment and no secondary object reference?

Reply:
Ok :)